### DISCUSSION TITLE
What do I loose with this simplified config?

### DISCUSSION THREAD

**Discussion by GordianDziwis** [ggandor/leap.nvim#87] [Q&A]
The config is:
```lua
local leap = require 'leap'
local target_windows = vim.tbl_filter(function(win)
    return vim.api.nvim_win_get_config(win).focusable
end, vim.api.nvim_tabpage_list_wins(0))
vim.keymap.set({ 'n', 'x', 'o' }, 'f', function()
    leap.leap {
        target_windows = target_windows,
        inclusive_op = false,
    }
end)
vim.keymap.set({ 'n', 'x', 'o' }, 'F', function()
    leap.leap {
        target_windows = target_windows,
        inclusive_op = true,
        offset = 1,
        match_last_overlapping = false,
    }
end)
```

The idea is that `f` does the motion to the left side of the two character match and `F` to the right side.
I think this needs less bandwidth than the default config: I do not need to have to think about, if I am going forward or backward, which window the pointer is or in which mode I am.

What am I missing?

**Comment by ggandor** [comment]
Automatic jump to the first target, mainly. See [FAQ](https://github.com/ggandor/leap.nvim#faq)/bidirectional search. Most frequently, you want to do something in the vicinity of the cursor, and it's _a lot more_ comfortable if you can just jump right there without having to enter a label character. But do whatever feels best.
